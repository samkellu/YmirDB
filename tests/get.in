get a
set a 123 22 44
get a
set b 12 a 44
get a
bye
