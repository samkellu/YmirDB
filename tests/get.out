==2804== Memcheck, a memory error detector
==2804== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2804== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==2804== Command: ./ymirdb
==2804== 
==2804== error calling PR_SET_PTRACER, vgdb might block
> no such key

> ok

> [123 22 44]

> ok

> [123 22 44]

> bye
==2804== 
==2804== HEAP SUMMARY:
==2804==     in use at exit: 0 bytes in 0 blocks
==2804==   total heap usage: 32 allocs, 32 frees, 1,872 bytes allocated
==2804== 
==2804== All heap blocks were freed -- no leaks are possible
==2804== 
==2804== For lists of detected and suppressed errors, rerun with: -s
==2804== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
